statsmodels/tsa/tests/test_tsa_indexes.py

```diff
@@ -710,6 +739,105 @@ def test_prediction_increment_pandas_dates_nanosecond():
     assert_equal(prediction_index.equals(desired_index), True)
 
 
+@pytest.mark.skipif(not has_range_index, reason='No RangeIndex')
+def test_range_index():
+    tsa_model.__warningregistry__ = {}
+
+    endog = pd.Series(np.random.normal(size=5))
+    assert_equal(isinstance(endog.index, pd.RangeIndex), True)
+    # Warning should not be given
```

Comment:
Does this mean that the call shouldn't emit any warnings, or that the test should suppress the warnings?  If the former, then better not to catch them.

Reply:
This means that prior to this PR (i.e. in master right now), a warning would be issued with any `RangeIndex` (including the default index for any pandas series for which an index was not explicitly given by the user). The test is that after the PR a warning is *not* issued for `RangeIndex`.